source/community.applications/usr/local/emhttp/plugins/community.applications/include/helpers.php

```
@@ -101,10 +101,40 @@ function writeGlobals($templates) {
 		unset($GLOBALS['templates']);
 		return;
 	}
-	$smallTemplates = duplicateArrayWithoutKeys($templates, ['Config','Network','MyIP','Shell','ExtraParams','PostArgs','CPUset','TailscaleEnabled','TailscaleIsExitNode','TailscaleDParams','TailscaleParams','TailscaleStateDir','TailscaleUserspaceNetworking']);
-	writeJsonFile(CA_PATHS['community-templates-info'],$smallTemplates);
-	writeJsonFile(CA_PATHS['community-templates-info-full'],$templates);
-	$GLOBALS['templates'] = $smallTemplates;
+	/* Write the templates as-is to the small cache — no second-pass strip.
+	   The server's slim feed already arrives without the heavy fields, and
+	   the full-feed fallback path explicitly writes the full cache itself
+	   via writeJsonFile() so we don't reiterate the array here. */
+	writeJsonFile(CA_PATHS['community-templates-info'], $templates);
```

PHPStan (2.1.54)

[error] Constant CA_PATHS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)